jmetal-core/src/main/java/org/uma/jmetal/util/experiment/component/ComputeQualityIndicators.java

```diff
@@ -39,7 +39,9 @@
  * @author Antonio J. Nebro <antonio@lcc.uma.es>
  */
 public class ComputeQualityIndicators<S extends Solution<?>, Result> implements ExperimentComponent {
+	
 
+  private static final String FILE_STRING = "File ";
```

Comment:
This factoring is clearly NOT one I would recommend. This is syntactical factoring. It has no semantic value. It just happens that we find this content several times, but none of the messages using it share enough semantic to motivate the creation of some common values beside `file`.

For example, instead of:
```java
JMetalLogger.logger.info("File " + file + " exist.");
...
JMetalLogger.logger.info("File " + file + " is a directory. Deleting directory.");
...
JMetalLogger.logger.info("File " + file + " is a file. Deleting file.");
```
we may rewrite like this:
```java
JMetalLogger.logger.info("Already existing file " + file);
...
JMetalLogger.logger.info("Deleting directory " + file);
...
JMetalLogger.logger.info("Deleting file " + file);
```
It is a fairly reasonnable rewritting, but there is clearly no reason to make any constant out of them. This is what I mean by being syntactical factoring (it only appears to be the same) and not semantic (it has no concrete reason to be the same).

Reply:
Done in 79e283c8